test/example/requirements.txt

```diff
@@ -10,3 +10,5 @@ cssselect
 coverage
 flake8
 tox
+# used for postgresql connection
+psycopg2
```

Comment:
can we still keep this in docker file? if it's here, not everyone that uses the docker file will have to install postgres. people can use this locally on sqlite without docker.

Reply:
ok. I did that on another commit. Please check it.